airbyte-integrations/connectors/source-stripe/integration_tests/expected_records.jsonl

```diff
@@ -23,6 +23,7 @@
 {"stream": "customers", "data": {"id": "cus_Kou8knsO3qQOwU", "object": "customer", "address": null, "balance": 0, "created": 1640123795, "currency": "usd", "default_source": "src_1MSID8EcXtiJtvvhxIT9lXRy", "delinquent": false, "description": null, "discount": null, "email": "edward.gao+stripe-test-customer-1@airbyte.io", "invoice_prefix": "CA35DF83", "invoice_settings": {"custom_fields": null, "default_payment_method": null, "footer": null, "rendering_options": null}, "livemode": false, "metadata": {}, "name": "edgao-test-customer-1", "next_invoice_sequence": 2, "phone": null, "preferred_locales": [], "shipping": null, "tax_exempt": "none", "test_clock": null, "updated": 1640123795}, "emitted_at": 1697627278435}
 {"stream": "customers", "data": {"id": "cus_NGoTFiJFVbSsvZ", "object": "customer", "address": {"city": "", "country": "US", "line1": "Street 2, 34567", "line2": "", "postal_code": "94114", "state": "CA"}, "balance": 0, "created": 1675160053, "currency": "usd", "default_source": "src_1MWGs8EcXtiJtvvh4nYdQvEr", "delinquent": false, "description": "Test Customer 2 description", "discount": null, "email": "user1.sample@zohomail.eu", "invoice_prefix": "C09C1837", "invoice_settings": {"custom_fields": null, "default_payment_method": null, "footer": null, "rendering_options": null}, "livemode": false, "metadata": {}, "name": "Test Customer 2", "next_invoice_sequence": 15, "phone": null, "preferred_locales": ["en-US"], "shipping": {"address": {"city": "", "country": "US", "line1": "Street 2, 34567", "line2": "", "postal_code": "94114", "state": "CA"}, "name": "Test Customer 2", "phone": ""}, "tax_exempt": "none", "test_clock": null, "updated": 1675160053}, "emitted_at": 1697627278439}
+{"stream":"payment_methods","data":{"id":"pm_1PYSZ3EcXtiJtvvhIos7UZSV","object":"payment_method","allow_redisplay":"limited","billing_details":{"address":{"city":null,"country":"CA","line1":null,"line2":null,"postal_code":"K1P 5L9","state":null},"email":"customer99900@example.com","name":"Customer 999","phone":null},"card":{"brand":"visa","checks":{"address_line1_check":null,"address_postal_code_check":"pass","cvc_check":"pass"},"country":"US","display_brand":"visa","exp_month":1,"exp_year":2026,"fingerprint":"X7e9fFB0r8MMcdo6","funding":"credit","generated_from":null,"last4":"4242","networks":{"available":["visa"],"preferred":null},"three_d_secure_usage":{"supported":true},"wallet":null},"created":1720010738,"customer":"cus_QPH7FKQ1vXX4N0","livemode":false,"metadata":{},"type":"card"},"emitted_at":1724941242506}
```

Comment:
Should we set one payment method for at least two customer to ensure that stream partitions are working in end-to-end?

Reply:
Test steps:
1. tried to update existing payment_method via [Update method](https://docs.stripe.com/api/payment_methods/update) => got an error: custoemr cannot be updated with this endpoint, try to detach/attach
2. made a [post detach](https://docs.stripe.com/api/payment_methods/detach) requests => successful
3. tried to [attach](https://docs.stripe.com/api/payment_methods/attach) payment_method from step 2 to another customer => got an error:

```json
{
    "error": {
        "message": "This PaymentMethod was previously used without being attached to a Customer or was detached from a Customer, and may not be used again.",
        "request_log_url": "https://dashboard.stripe.com/test/logs/req_u3sulw0pAzAzX8?t=1725528614",
        "type": "invalid_request_error"
    }
}
```
Conclusion: Payment cannot be set for 2 customers.

Reply:
ah, ok, will add another one

Reply:
added in https://github.com/airbytehq/airbyte/pull/44891/commits/90fa9c83b8c8a9f464f8075892d82ffb0e606d4d